architecture/kubernetes_infrastructure.adoc

```diff
@@ -9,43 +9,41 @@
-
-The controller manager server watches etcd for changes to replication controller objects and then uses the API to enforce the desired state.
+The controller manager server watches [sysitem]#etcd# for changes to replication controller objects and then uses the API to enforce the desired state.
 
 == Node
+A node is a system providing the run-time environments for containers. Each node in a Kubernetes cluster is managed by the link:#master[master]. Nodes have the services necessary for this management and to run pods, including Docker, a link:#kubelet[kubelet], and a link:#service-proxy[service proxy].
 
-Kubernetes validates the node with health checks and ignores it until these health checks are passed. Note that Kubernetes continues to check nodes until they are valid.
+Nodes are created from your cloud provider, or from your own physical or virtual systems. Kubernetes interacts with node objects that are a representation of those nodes. The master uses the information from node objects to validate nodes with health checks. A node is ignored until the health checks are passed, however the master continues to check nodes until they are valid.
 
-Note that kubernetes does not create nodes; rather, a node is instead created by your cloud provider, or from your physical or virtual machines. 
+See the link:../using_openshift/managing_nodes.html[Managing Nodes] topic for more information on node management for administrators using the CLI.
 
+[[kubelet]]
 === Kubelet
-
-Each node has a kubelet that updates the node as specified by a container manifest; a YAML file that describes a pod. The kubelet uses a set of manifests to ensure that the containers described start and continue to run. A sample manifest can be found https://cloud.google.com/compute/docs/containers/container_vms#container_manifest[here].
+Each node has a kubelet that updates the node as specified by a container manifest; a YAML file that describes a pod. The kubelet uses a set of manifests to ensure that the containers described start and continue to run. A sample manifest can be found in the  https://cloud.google.com/compute/docs/containers/container_vms#container_manifest[Kubernetes documentation].
 
 There are a number of ways that a container manifest can be provided to a kubelet:
 
-* A file path on the command line. The file is checked every twenty seconds.
-* A HTTP endpoint passed on the command line. The endpoint is checked every twenty seconds.
-* The kubelet watches an etcd server, such as `[filename]#/registry/hosts/$(hostname -f)#`. Any changes are noticed and acted upon.
+* A file path on the command line. The file is checked every 20 seconds.
+* An HTTP endpoint passed on the command line. The endpoint is checked every twenty seconds.
+* The kubelet watches an [sysitem]#etcd# server, such as `[filename]#/registry/hosts/$(hostname -f)#`. Any changes are noticed and acted upon.
 * The kubelet listens for HTTP and respond to a simple API to submit a new manifest.
 
```

Comment:
[suggestions]
A file path on the command that is checked every twenty seconds.
An HTTP endpoint passed on the command that is checked every twenty seconds.
A kubelet watching an etcd server, such as /registry/hosts/$(hostname -f), and acting on any changes that are recognized.